django_project_base/country_holidays.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+
+import requests
+from django.conf import settings
+from django.core.cache.backends.filebased import FileBasedCache
+from rest_framework import status
+
+country_holidays_storage: FileBasedCache = FileBasedCache(settings.DJANGO_PROJECT_BASE_COUNTRY_HOLIDAYS_CACHE_FOLDER, {
```

Comment:
Tako kot pri fileuploadfieldu tudi tu ne vidim potrebe po uvajanju novih, posebnih, storage mehanizmov. Uporabi pač django cache - zakaj ne?

Reply:
spremenjeno na cache

Reply:
Dodan test in osnova dokumentacija